packages/events/ReactGenericBatching.js

```diff
@@ -52,8 +52,8 @@ export function batchedUpdates(fn, bookkeeping) {
   }
 }
 
-export function interactiveUpdates(fn, a, b) {
-  return _interactiveUpdatesImpl(fn, a, b);
+export function interactiveUpdates(fn, a) {
+  return _interactiveUpdatesImpl(fn, a);
 }
```

Comment:
This possibly affects other renders 😨

Reply:
Have you checked whether it does? `ReactGenericBatching` is only accessible in this repository.

Reply:
This should not affect other renders. ReactGenericBatching.interactiveUpdates is only used inside of ReactDOMEventListener.